ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -762,6 +762,9 @@ tests:
   run_if_changed: /agent/
   steps:
     cluster_profile: equinix-ocp-metal
+    env:
+      DEVSCRIPTS_CONFIG: |
+        NETWORK_TYPE=OVNKubernetes
```

Comment:
I don't think this will work... either this setting of `DEVSCRIPTS_CONFIG` will override the one in `ci-operator/step-registry/agent/e2e/compact/ipv4/`, or that one will override this one. They definitely won't get merged.

Probably you should add `NETWORK_TYPE` there

Reply:
The problem with both this patch and changing it in the step registry is that it changes it for all versions, so pre-4.15 release-* branches won't test OpenShiftSDN any more.

Reply:
> The problem with both this patch and changing it in the step registry is that it changes it for all versions, so pre-4.15 release-* branches won't test OpenShiftSDN any more.

Won't earlier releases use the release specific file https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-release-4.14.yaml#L697-L701

Reply:
> I don't think this will work... either this setting of DEVSCRIPTS_CONFIG will override the one in ci-
> operator/step-registry/agent/e2e/compact/ipv4/, or that one will override this one. They definitely 
> won't get merged.
You're right, redoing this...